tests/integration/test_stages.py

```diff
@@ -1,3 +1,4 @@
+# pylint:disable=protected-access,no-self-use
```

Comment:
@jbzdak please don't disable checks for a whole file - either don't use protected access, or suppress one-by-one